force-app/main/default/classes/countOpenTasks_TriggerHandler.cls

```diff
@@ -0,0 +1,106 @@
+/**
+ * @description       :
+ * @author            : ChangeMeIn@UserSettingsUnder.SFDoc
+ * @group             :
+ * @last modified on  : 02-01-2023
+ * @last modified by  : ChangeMeIn@UserSettingsUnder.SFDoc
+ **/
+public with sharing class countOpenTasks_TriggerHandler {
+  public static void handlerAfterInsert(List<Task> newTasks) {
+    processTasks(newTasks);
+  }
+
+  public static void handlerAfterUpdate(
+    List<Task> oldTasks,
+    List<Task> newTasks
+  ) {
+    processTasks(oldTasks);
+  }
+
+  public static void handlerAfterDelete(List<Task> oldTasks) {
+    processTasks(oldTasks);
+  }
+
+  private static void processTasks(List<Task> tasks) {
+    Set<Id> accountIds = new Set<Id>();
+    ID Recordtype = [
+      SELECT Id
+      FROM RecordType
+      WHERE Name = 'United States'
+      LIMIT 1
+    ]
+    .Id;
+
+    for (Task t : tasks) {
+      if (t.WhatId != null) {
+        accountIds.add(t.WhatId);
+      }
+    }
+
+    List<Account> rltdAccounts = [
+      SELECT
+        Id,
+        RecordTypeID,
+        Type,
+        Open_Tasks_Email__c,
+        Open_Tasks_Call__c,
+        Open_Tasks_Meeting__c,
+        Open_Tasks_Other__c,
+        (
+          SELECT Id, Type, Status, WhatId
+          FROM Tasks
+          WHERE WhatId IN :accountIds AND Status != 'Completed' AND Type != NULL
+        )
+      FROM Account
+      WHERE
+        Id IN :accountIds
+        AND Type = 'Prospect'
+        AND RecordTypeID = :Recordtype
+    ];
+
+    Integer openEmailTask = 0;
+    Integer openCallTask = 0;
+    Integer openMeetingTask = 0;
+    Integer openOtherTask = 0;
+
+    Map<Id, Account> updatedAccounts = new Map<Id, Account>();
```

Comment:
accountsToUpdateMap

updatedAccounts means, they have been already updated, also it suggests a List, not a map